charts/argo-workflows/templates/crds/argoproj.io_workflowtaskresults.yaml

```diff
@@ -116,7 +116,7 @@ spec:
                           required:
                           - key
                           type: object
-                        blob:
+                        blobNameFormat:
```

Comment:
The upstream CRD still has `blob` [here](https://github.com/argoproj/argo-workflows/blob/cd2e4e564960d6c5f1a772d9d27672a08b3a6bcf/manifests/base/crds/minimal/argoproj.io_workflowtaskresults.yaml#L111-L112).  Please let me know where you are seeing the `blobNameFormat` in v3.4.10 upstream.

Reply:
Are we waiting on an upstream change I'm missing?  I've looked through [here](https://github.com/search?q=repo%3Aargoproj%2Fargo-workflows+blobNameFormat&type=code) and didnt' find anything.

Reply:
sorry I am absolute novice here. I was looking at doc [here](https://github.com/argoproj/argo-workflows/blob/cd2e4e564960d6c5f1a772d9d27672a08b3a6bcf/sdks/java/client/docs/IoArgoprojWorkflowV1alpha1AzureArtifactRepository.md?plain=1#L12), and also had issue from #2200.

Reply:
oops, sorry I think I was confusing AzureArtifactRepository and AzureArtifact. No need for this line